packages/upgrade/src/upgrades.js

```diff
@@ -664,6 +664,36 @@ export const upgrades = [
           });
         },
       },
+      {
+        name: 'ibm-products-update-pdlc-status',
```

Comment:
Should this be more clear for those that might not know what PDLC is?

```suggestion
        name: 'ibm-products-rename-imports-to-preview',
```